INTITA/Modules/Profile/ProfileCoordinator.swift

```diff
@@ -38,6 +38,24 @@ class ProfileCoordinator: Coordinator {
     func showLoginScreen() {
         navigationController.popViewController(animated: true)
     }
+    
+    func displaySettingsProfileScreen() {
+        
+        let settingScreenCoordinator = SettingsProfileCoordinator(navigationController: navigationController)
+        
+        
+        settingScreenCoordinator.start()
+    }
+    
+    func removeCoordinator(_ coordinator: Coordinator) {
```

Comment:
We already don't need this function, remove it please

Reply:
fixed